lib/reply.js

```diff
@@ -88,10 +88,20 @@ Reply.prototype.hasHeader = function (key) {
   return this._headers[key.toLowerCase()] !== undefined
 }
 
+Reply.prototype.removeHeader = function (key) {
+  // Node.js does not like headers with keys set to undefined,
+  // so we have to delete the key.
+  delete this._headers[key.toLowerCase()]
+}
+
 Reply.prototype.header = function (key, value) {
   var _key = key.toLowerCase()
-  // https://tools.ietf.org/html/rfc7230#section-3.2.2
+
+  // default the value to ''
+  value = value || ''
```

Comment:
We should add a more strict check, otherwise also `false`, `null` and `0` will be changed.